README.md

```diff
@@ -6,62 +6,40 @@ This repository contains some helper/wrapper scripts to make working with GN eas
 
 ### macOS / Linux
 
-```bash
-git clone https://github.com/MarshallOfSound/electron-gn-scripts.git
-cd electron-gn-scripts
-yarn
-# You could also use `npm install` here
-# You should probably add this to your path in your `.zshrc` or `.bashrc`
-export PATH="$PATH:$(pwd)/nix"
+```sh
+$ git clone https://github.com/MarshallOfSound/electron-gn-scripts.git
+$ cd electron-gn-scripts
+$ export PATH="$PATH:/path/to/electron-gn-scripts/nix"
```

Comment:
I liked the copy-pastability of the `$(pwd)`.  Is there a reason that was killed?

Reply:
Yes, the comment about 'you should probably add this to your dotfile' -- adding `PATH="$PATH:$(pwd)/nix"` to a dotfile doesn't help anyone.

I've split it into two separate lines in 54f9f6de73a446ccdf2e787d32968b192afd1175 so that it's still copy-pastable